src/components/products/product-card/ProductCard.scss

```diff
@@ -0,0 +1,138 @@
+.product-card {
```

Comment:
**note:** there are several naming guides for CSS classes. Several from the most popular [BEM](https://getbem.com/) and [SMACSS](https://smacss.com/)

Reply:
Helpful. Revising it now